core/file_handler.py

```
@@ -62,7 +62,10 @@ def receive_data_chunk(self, raw_data, start):
             mime = magic.from_buffer(raw_data, mime=True)
             if mime not in self.ACCEPTED_FILE_UPLOAD_MIME_TYPES:
                 self.abort()
-                raise UnacceptableMimeTypeError(f"Unsupported file type: {mime}")
+                # Raise StopUpload signals the django multipartparser to exhaust the stream and stops the upload
+                exception = UnacceptableMimeTypeError(f"Unsupported file type: {mime}")
+                logger.error(exception)
```

We now log the error since the django multipartparser will swallow this exception.